### PR TITLE
[update] Gracefully terminate on SIGINT

### PR DESCRIPTION
**Pull request by actualdankcoder**
#3

### CODE REVIEW

**Comment by actualdankcoder** [comment]
untested because i haven't setup next locally, will try later

**Comment by actualdankcoder** [comment]
@Shinyzenith does it have to be atexit or does signal work as well, i don't have much idea where this will be in effect

**Comment by Shinyzenith** [comment]
> @Shinyzenith does it have to be atexit or does signal work as well, i don't have much idea where this will be in effect

Anything works as long as we call NextCore.display.terminate / NextCore.destroy on sigint

File: libnext/backend.py [hunk 1]
```diff
@@ -136,7 +140,11 @@ def __init__(self) -> None:
         self.backend.destroy()
         self.display.destroy()
 
-    # Properties
+    # Cleanup on SIGNINT
+    def cleanup(self) -> None:
```
Comment (Shinyzenith) [inline libnext/backend.py:144]: cleanup logic is in the init function rn as display.run is a blocking call, I will split it into a different function

**Pull request closed without merging**